tests/lib/rss.test.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * RSS Library Test
+ */
+
+const fs = require('fs')
+const rss = require('../../lib/rss')
+const mockdate = require('mockdate')
+
+describe('lib.rss', () => {
+  let rssResponse
+
+  beforeEach(() => {
```

Comment:
Sure if the object will not be manipulated then use `beforeAll`, but we will definitely manipulate it in the future.

Reply:
Alright, left this as-is and changed `afterAll` to `afterEach` so they match.